Sources/Mini/Dispatcher.swift

```diff
@@ -100,6 +85,15 @@ public final class Dispatcher {
         )
         return registerInternal(subscription: subscription)
     }
+    
+    private func getNewSubscriptionId() -> Int {
```

Comment:
This method is not thread safe. Data races may occur and crash. Switch back to atomic operations if possible.

Reply:
Ok, I have added an Atomic class to control this value in a safe way. It should be similar to the way you had it implemented before.